frontend/src/views/FormContainer.tsx

```diff
@@ -10,28 +10,44 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-import { Banner } from 'core/banner/Banner';
-import { Text } from 'core/text/Text';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
 import PropTypes from 'prop-types';
 import React from 'react';
 import styles from './FormContainer.module.css';
```

Comment:
Remove this import and move this CSS module style to `makeStyles`

Reply:
done